docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+:::info Following References 
+You *can* track `Line` or `dlge` (dialogue) files via the <Tooltip text={<strong>Reference list</strong>} tooltip="A tab that helps you trace how your file connects to others, showing which files use it or by which ones it's getting.. used" /> to their respective <Tooltip text={<strong>LOCR Files</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> too, but using my search method, it'll be highly unlikely you haven't located your <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />
+:::
+
+[**Continue with you LOCR file**](#pt-2-get-your-string)
+
+
+
+---
+
+## Advanced Search Guide AI
+
+When searching for text beyond suits, the **Advanced Search Tab** in GlacierKit provides extended capabilities.
+
+### How to Access Advanced Search:
+1. **Click** the **Advanced Search Tab** (magnifying glass icon on the left).
+
+### Setting Up Your Search:
+- **Select**: Only check the `chunk0` box.
+- **Filter**: Enable **"localisation"** to narrow results specifically to text files.
+
+   :::note
+   - *`Chunk0` is the primary source for text data.*
+   - *Using the "localisation" filter speeds up your search.*
+   :::
+
+### Searching Effectively:
+*Remember, searches are case-sensitive.* 
+- **Try variations** of your text to cover all bases:
+   - "Blood Money"
+   - "blood money"
+   - "BLOOD MONEY"
+   - "bloodmoney"
+
+   :::tip Multiple Searches
+   You can run several searches at once. Watch the task bar at the bottom for updates on their progress.
+   :::
+
+### Finding the Right File:
+- Look specifically for **LOCR Files**. 
+- **Skip**: `Line` or `dlge` files; they aren’t relevant for localisation overrides.
+
+   :::info Reference Tip
+   If you come across `Line` or `dlge` files, use the **Reference List** tab to trace them back to the correct **LOCR File**.
+   :::
+
+---
+
+
+
+#### All languages
+These are all the language codes for the <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" /> that can be used for <Tooltip text={<strong>'localisationOverrides'</strong>} tooltip="A way to replace or modify in-game text, like names or descriptions, with your own custom text. It allows you to customize how text appears without altering the core game files." />;
+- `"english"`
+- `"french"`
+- `"italian"`
+- `"german"`
+- `"spanish"`
+- `"russian"`
+- `"chineseSimplified"`
+- `"chineseTraditional"`
+- `"japanese"`
+
+[Return to Adding languages](#5-optionally-add-more-languages)
+
+---
+
+<details>
+<summary>**Glossary**</summary>
+### Glossary 
```

Comment:
You don't exactly need a glossary when literally every key word possible is in bold, italic and has a tooltip